rs/bitcoin/kyt/tests/tests.rs

```diff
@@ -27,11 +27,11 @@ const MAX_TICKS: usize = 10;
 // by a small margin. Namely, the universal_canister itself would consume
 // some cycle for decoding args and sending the call.
 //
-// The number 7_000_000 is obtained empirically by running tests with pocket-ic
+// The number 42_000_000 is obtained empirically by running tests with pocket-ic
 // and checking the actual consumptions. It is both big enough to allow tests to
 // succeed, and small enough not to interfere with the expected cycle cost we
 // are testing for.
-const UNIVERSAL_CANISTER_CYCLE_MARGIN: u128 = 7_000_000;
+const UNIVERSAL_CANISTER_CYCLE_MARGIN: u128 = 42_000_000;
```

Comment:
This change LGTM. But I wonder why it went from 7M to 42M. Here is my rough calculation, please help to see where I was doing it wrong:

The test was measuring cycle difference before/after call on the Universal Canister, not the KYT canister. The `expected_cost` is what is charged by the KYT canister, so it shouldn't change. The` actual_cost - expected_cost` would be for the Universal Canister to processing one ingress, do some internal computation, make an inter-canister call, receives its reply and forward the reply back to the original caller. There are 2 invocations, so base charge would be 10M. The rest would be computation charge, which I think is increased by 2.5x? That would be 6.5M x 2.5 = 16.25M. So the numbers don't seem to add up to 42M.

Reply:
Yeah, I know it's a bit weird. Are you sure you had used the "minimum" in the past to set at margin? I added printing of the two values in the assert and I saw them differ by ~40M, hence I went with 42M. Feel free also to double check the values yourself (you can make the test fail locally and see what it prints).

Reply:
I'll go ahead and merge since we would like this change to be merged. Maybe we can figure out if it can get tighter asynchronously in a follow up.